### ISSUE TITLE
The scale of different values in Figure 7 doesn't match

### ISSUE THREAD

**Issue by BaohaoLiao** [ylsung/Ladder-Side-Tuning#10]
Hi,

in Figure 7 at this [newest version](https://arxiv.org/pdf/2206.06522.pdf), the bar gap between 83.4, 83.5 and 83.6 are weird. You might annotate the wrong number there.

In addition, I have a question about Table 1. You use LST on T5-large to increase the training memory for fairly comparing to other methods, like Adapters and LoRA. Do you have the results for LST on T5-base with a smaller reduction factor? In this way, you can also increase the training memory and all methods use the same backbone.

**Comment by ylsung** [comment]
I rounded the numbers in the annotation, and the true value in the figures are 83.40, 83.47, and 83.64, so that's why 83.5 looks closer to 83.4 and further from 83.6.

Regarding the second question, we did try to use more memory by reducing the reduction factor but the scaling results are not good. We hypothesize that this is because the initialization of the side network is not strong enough, where I assume using distillation with pre-training data would be better.
